patch/core/book.js

```diff
@@ -26,19 +23,20 @@ module.exports = {
 			} else if (item.includes('crystal')) {
 				item = 'crystal';
 			}
-
+			
 			// set clue text based on actor type
 			if (actor === 'Death') {
 				clues.push('Death guards\n' + item);
 			} else if (actor === 'Camilla') {
 				clues.push('Camilla\ndefends\n' + item);
 			} else if (actor === 'merchant') {
-				clues.push(item + '\nfor sale in\n' + location);
+				//ignore basic merchants to make clues more relevant
+				//clues.push(item + '\nfor sale in\n' + location);
 			} else if (actor === 'sacred flame') {
 				clues.push(item + '\nis hidden\non Dabi\'s Path');
 			} else if (actor === 'orb') {
 				clues.push(item + '\nhidden in\n' + location + ' orb');
-			} else if (actor === 'crystal dude') {
+			} else if (actor === 'crystal dude' || actor === 'secret merchant') {
```

Comment:
Given that secret merchants require garlic, they aren't technically free. It's worth it (and fun) to note the necessity of garlic in a separate text for secret merchants

Reply:
couldn't think of anything good. how would you phrase it?